.travis.yml

```diff
@@ -6,8 +6,8 @@ git:
 
 language: python
 
-python:
-  - 3.6
+python: 
```

Comment:
```suggestion
python:
```

Reply:
Still some missing whitespace.

Reply:
Got it, I'm gonna wait to see what we decide re: url_encoding before merging this in, in case we decide to modify that functionality